demo_code/tests/test_playwright.py

```diff
@@ -101,12 +98,10 @@ def test_destination_options_have_cruises(page: Page, mock_functions_env, live_s
 
     for page_cruise in page_cruises:
         assert page_cruise.text_content() in cruises
```

Comment:
```suggestion
    expect(page.locator(".list-group-item")).to_contain_text(cruises)
```

this should also work in theory, but needs to be tested before. then you can also remove the line 97.

Reply:
That results in errors like:

>       result = next(iter(done)).result()
E       playwright._impl._api_types.Error: Error: strict mode violation: locator(".list-group-item") resolved to 2 elements:
E           1) <a href="/cruise/3" class="list-group-item list-gro…>The Cold Planets Expedition</a> aka get_by_role("link", name="The Cold Planets Expedition")
E           2) <a href="/cruise/5" class="list-group-item list-gro…>The Grand Solar System Tour</a> aka get_by_role("link", name="The Grand Solar System Tour")
E